src/app/components/MediaLoader/configs/portraitClipMedia.ts

```diff
@@ -23,77 +22,102 @@ export default ({
+
+  if (!firstBlock) {
+    return {
+      mediaType: 'video',
+      playerConfig: basePlayerConfig,
+      showAds: false,
+      orientation: 'portrait',
+    };
+  }
+
+  const portraitClipMediaBlocks: PortraitClipMediaBlock[] = filterForBlockType(
+    blocks,
+    'portraitClipMedia',
+    { multiple: true },
+  ) as PortraitClipMediaBlock[];
+
+  const playlistItems: PlaylistItem[] = portraitClipMediaBlocks.map(block => {
+    const { model } = block;
+    const { video, images } = model;
+    const version = video?.version;
+
+    return {
+      versionID: version?.id,
+      kind: version?.kind || 'programme',
+      duration: moment.duration(version?.duration || 'PT0S').asSeconds(),
+      embedRights: video?.isEmbeddingAllowed ? 'allowed' : undefined,
+      vpid: video?.id,
+      serviceID: version?.territories?.[0],
+      title: video?.title,
+      guidance: version?.guidance,
+      territories: version?.territories,
+      images,
+    };
+  });
 
   const showAds = shouldDisplayAds({
     adsEnabled,
     showAdsBasedOnLocation,
-    duration: rawDuration,
+    duration: playlistItems[0]?.duration ?? 0,
   });
 
-  const embeddingAllowed = video?.isEmbeddingAllowed ?? false;
+  if (showAds) {
+    playlistItems.unshift({ kind: 'advert' });
+  }
+
+  const captionBlock = getCaptionBlock(blocks, 'live');
+  const caption =
+    captionBlock?.model?.blocks?.[0]?.model?.blocks?.[0]?.model?.text;
 
   const holdingImageURL = buildIChefURL({
-    originCode,
-    locator,
+    originCode:
+      firstBlock?.model?.images?.[1]?.source?.replace('Image', '') ?? '',
+    locator: firstBlock?.model?.images?.[1]?.urlTemplate ?? '',
     resolution: DEFAULT_WIDTH,
   });
 
-  const items: PlaylistItem[] = [{ versionID, kind, duration: rawDuration }];
-
-  if (showAds) items.unshift({ kind: 'advert' });
-
-  const externalEmbedUrl = getExternalEmbedUrl({ id, versionID, lang });
+  const externalEmbedUrl = getExternalEmbedUrl({
+    id,
+    versionID: firstBlock?.model?.video?.version?.id,
+    lang,
+  });
 
   return {
-    mediaType: type || 'video',
+    mediaType: firstBlock?.model?.type ?? 'video',
     playerConfig: {
       ...basePlayerConfig,
       ...(externalEmbedUrl && { externalEmbedUrl }),
       autoplay: true,
       playlistObject: {
-        title,
+        title: firstBlock?.model?.video?.title,
         summary: caption || '',
         holdingImageURL,
-        items,
-        ...(guidanceMessage && { guidance: guidanceMessage }),
-        ...(embeddingAllowed && { embedRights: 'allowed' }),
+        items: playlistItems,
       },
       ui: {
         ...basePlayerConfig.ui,
-        ...(type === 'audio' && AUDIO_UI_CONFIG),
+        swipable: {
+          enabled: true,
+          direction: 'Y',
+        },
+        controls: {
+          enabled: true,
+          includeNextButton: true,
+          includePreviousButton: true,
+        },
+        ...(firstBlock?.model?.type === 'audio' && AUDIO_UI_CONFIG),
```

Comment:
Could this be moved above the `swipable` and `controls` objects. Generally its best to put spreads at the top and then the others below so that they overwrite anything in the spreads correctly.